src/main/java/uk/gov/hmcts/reform/preapi/controllers/BookingController.java

```diff
@@ -53,8 +53,15 @@ public BookingController(final BookingService bookingService) {
     @Parameter(
         name = "caseReference",
         description = "The Case Reference to search for",
+        schema = @Schema(implementation = String.class),
         example = "1234567890123456"
     )
+    @Parameter(
```

Comment:
Do we want to expose the parameter to control the size of pages as well?

Reply:
sure, what is it called? :S

Reply:
I think its `size`